LICENSE

```diff
@@ -0,0 +1,17 @@
+Portions of this repository are available under one of the following licenses
```

Comment:
This file really doesn't say anything, I'm not sure there's value in including it. The tl'dr is `there's different licenses, look at them` which someone would do in the absence of this file and was looking to find out.

Reply:
> The tl'dr is there's different licenses, look at them

I agree, though I think explicitly saying it could be helpful. I tried not to be much more long-winded than that. :-)

The second section on, about making new contributions under the first license listed is new, and not something we've really discussed before.

Reply:
> New files should be made under the first license listed in the appropriate package.xml file

I didn't see that. I don't like the policy of asserting licenses on people. I rotate licenses from BSD to GPL depending on what it is and where its going. We can _say_ that, but in practice, I can say I don't plan on following it